editor/src/components/editor/store/editor-state.spec.ts

```diff
@@ -127,9 +132,28 @@ describe('modifyUnderlyingTarget', () => {
       import { jsx, Rectangle } from 'utopia-api'
       export var Card = (props) => {
         return (
-          <div>
-            <div data-thing='a thing' />
-            <Rectangle />
+          <div style={{ ...props.style }}>
+            <div
+              style={{
+                position: 'absolute',
+                left: 0,
+                top: 0,
+                width: 50,
+                height: 50,
+                backgroundColor: 'red',
+              }}
+              data-thing='a thing'
+            />
+            <Rectangle
+              style={{
```

Comment:
this style prop is new here, right? where is this coming from?

Reply:
The tests were previously using a different version of the complex multifile project for no apparent reason, so I unified that to ensure we kept the two inline (as I was updating the sample project and wondering why it didn't break multifile tests)